src/components/Books.js

```diff
@@ -29,20 +29,48 @@ const useGetBook = (bookId) => {
     };
 };
 
-const updateBookStatus = async (bookId, checkedOutBy) => {
+const updateBookToReturned = async (bookId, userProfile) => {
+    const event = {
+        timestamp: new Date(Date.now()).toJSON(),
+        name: userProfile.name,
+        email: userProfile.email
+    };
     return await documentClient.update({
         TableName: process.env.REACT_APP_BOOK_TABLE,
         Key: {
             bookId
         },
-        UpdateExpression: "set checkedOutBy=:l",
+        UpdateExpression: "set checkedOutBy=:l, returnEvents=list_append(if_not_exists(returnEvents, :emptyList), :returnEvents)",
         ExpressionAttributeValues: {
-            ':l': checkedOutBy
+            ':l': null,
+            ':emptyList': [],
+            ':returnEvents': [event]
```

Comment:
Should this attribute be renamed to avoid ambiguity with the other returnEvents value?

Reply:
Not sure I follow. Rename `:returnEvents` to something else? Or are you referring to another param?

Reply:
Yes, that was my thought. Having both `returnEvents` and `:returnEvents` in the code seems confusing